timeless/employees/models.py

```diff
@@ -23,18 +23,17 @@ class Employee(TimestampsMixin, DB.Model):
     pin_code = DB.Column(DB.Integer, unique=True, nullable=False)
     comment = DB.Column(DB.String)
     company_id = DB.Column(DB.Integer, DB.ForeignKey("companies.id"))
+    """
+    @todo #348:30min In Employee model append attribute 
     role_id = DB.Column(DB.Integer, DB.ForeignKey("roles.id"), nullable=True)
+    after the foreign key column "role_id" will be implemented in the "employees" table of the database.
```

Comment:
@VladimirRyzhkov Hm, What is wrong with `role_id`? As I can see it's already implemented

Reply:
@vladarefiev  Vlad, I mean we have no ```role_id``` in the table and could't select it yet: https://github.com/timelesslounge/timelessis/blob/master/migrations/versions/2019-01-29T092044.py
May be there is another place where it can be seen?

Reply:
@vladarefiev Vlad, sorry, it's my misunderstandings. Now I see.